input/includes/menu.xml

```diff
@@ -16,6 +16,9 @@
       <b class="caret"></b>
     </a>
     <ul class="dropdown-menu">
+      <li>
+        <a href="artifacts.html">Artifacts Summary</a>
```

Comment:
Should be 'Artefacts Summary'.

Reply:
Amended as per resolution, noting title of page cannot be amended and remains as 'Artifacts Summary'

Reply:
It can - in ig.xml, and also to ensure that the Table of Contents for the IG subpages can be listed as per the menu areas which also is reflected in the breadcrumbs navigation, as per the JIRA resolution.